scss/atoms/typography/_typography.scss

```diff
@@ -110,7 +110,7 @@ strong {
 
 code {
 	background: map-get($typography-code,background);
-	font-size: em( 16px );
-	padding: em( 4px, 16px ) em( 8px, 16px );
+	font-size: rem( 16px );
+	padding: rem( 4px, 16px ) rem( 8px, 16px );
```

Comment:
Padding should probably be em not rem. Again a point we can discuss during next meeting. Padding is meant to be space between border and text to make text look better and readable, it makes sense to decide padding in scale of current element font size rather than root element font size.